sourced/ml/core/algorithms/token_parser.py

```diff
@@ -76,6 +80,23 @@ def max_token_length(self, value):
             raise ValueError("max_token_length must be greater than 0 - got %d" % value)
         self._max_token_length = value
 
+    @property
+    def use_nn(self):
+        return self._use_nn
+
+    @use_nn.setter
```

Comment:
Let's simplify our life and not allow changing `use_nn`

Reply:
Ok let's do that